### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by barnabasbartha**

### CODE REVIEW

File: src/main/server/player/server-player-component.ts [hunk 1]
```diff
@@ -137,13 +142,19 @@ export class ServerPlayerComponent {
...
       this.collisionPhysics.updatePosition(playerId, position.x, position.y);
-      // TODO: Check lava distance and kill if closer
       this.ammo.pickupInRectangle(
          playerId,
          position.x,
          position.y,
          ServerConfig.PLAYER_WIDTH,
          ServerConfig.PLAYER_HEIGHT,
       );
+      if (this.lavaDistance2(position) <= ServerConfig.LAVA_RADIUS * ServerConfig.LAVA_RADIUS * 0.9) {
```
Comment (dolanmiu) [inline src/main/server/player/server-player-component.ts:152]: should this not check if the player's `x,y` is less than the circumference of the circle?

https://www.w3resource.com/javascript-exercises/javascript-basic-exercise-120.php

File: src/main/server/player/server-player-component.ts [hunk 2]
```diff
@@ -137,13 +142,19 @@ export class ServerPlayerComponent {
...
+         this.resetPlayer(playerId);
+      }
+   }
+
+   private lavaDistance2(position: Vector): number {
+      return position.x * position.x + position.y + position.y;
```
Comment (dolanmiu) [inline src/main/server/player/server-player-component.ts]: `x^2 + 2y`?

Is this correct?

Comment (barnabasbartha) [reply src/main/server/player/server-player-component.ts]: Yeah a typo

File: src/main/server/player/server-player-component.ts (hunk 1 above)
Comment (barnabasbartha) [reply src/main/server/player/server-player-component.ts:152]: It does exactly this one doesn't it? Apartm from the typo. X2+y2<r2 cos the other point is 0,0

**Pull request merged**